_layouts/security.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: base
+---
+
+<div class="security-page grid-wrapper">
+    <div class="grid__item width-12-12">
+        <h1>Reporting of CVEs and Security Issues</h1>
+        <h2 class="page-subtitle">The Wild<strong>Fly</strong> community and our sponsor, Red Hat, take security bugs very
+            seriously
+        </h2>
+        <p>We aim to take immediate action to address serious security-related problems that involve our projects.
+            Note that we will only fix such issues in the most recent major release of Wild<strong>Fly</strong>.</p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-red">
+        <h2>Reporting of Security Issues</h2>
+        <p>When reporting a security vulnerability it is important to not accidentally broadcast to the world that the
+            issue exists, as this makes it easier for people to exploit it. The software industry uses the term
+            <a href="https://www.redhat.com/en/blog/security-embargoes-red-hat">embargo</a> to describe the time a security
+            issue is known internally until it public knowledge.
+        </p>
+        <p>
+            Our preferred way of reporting security issues in Wild<strong>Fly</strong> and its related projects are listed below.
+        </p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-teal">
+        <h2>Email the mailing list</h2>
+        <p>The list at <a href="mailto:XXX@redhat.com">XXX@redhat.com</a> is the preferred mechanism for outside users
+            to report security issues. This A member of the Wild<strong>Fly</strong> team will open the required issues.</p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-yellow">
+        <h2>Open a Jira issue</h2>
+        <p>If you are an experienced contributor, and choose to open a Jira issue in the correct project at
+            <a href="https://issues.redhat.com">https://issues.redhat.com</a>, make sure to set the
+            <strong>Security Level</strong> field to <strong>Embargoed Security Issue</strong>. This keeps it private.
+        </p>
+        <p>
+            Note that if you do open a Jira issue with the Security Level set to Embargoed Security Issue, you will not be able
+            to view the issue due to our security policy.
+        </p>
+    </div>
```

Comment:
I'm doubtful as to whether we should have this section. The simple policy a bunch of projects added only recommends email reporting, as does https://quarkus.io/security/ and https://www.keycloak.org/security.html.

If someone reads this and has two options it greatly increases the chances of them doing it wrong. If someone doesn't read this, it doesn't matter what it says. :)

Reply:
@bstansberry I was in doubt whether to include this for the same reasons. I can remove it